rs/bitcoin/ckbtc/minter/tests/tests.rs

```diff
@@ -1771,10 +1811,12 @@ fn test_ledger_memo() {
     let memo = res.transactions[0].mint.clone().unwrap().memo.unwrap();
     assert_eq!(
         ckbtc.kyt_provider,
-        res.transactions[0].mint.clone().unwrap().to.owner.into()
+        res.transactions[0].mint.clone().unwrap().to.owner.into(),
     );
     let decoded_data = minicbor::decode::<MintMemo>(&memo.0).expect("failed to decode memo");
     assert_eq!(decoded_data, MintMemo::Kyt);
+    // Make sure only deposit fee is transferred to kyt_provider
```

Comment:
On that note
> We do persist event logs and replay them during upgrades.

It might be useful to have a unit test, similar to what is done for ckETH [here](https://github.com/dfinity/ic/blob/aeb2450d0d6993e3733327f8dfc00d444c4f7575/rs/ethereum/cketh/minter/src/state/audit/tests.rs#L24) to ensure that we can replay events and that we end up in the same state